Gameplay/entities/crystals/CrystalExplosion.cpp

```diff
@@ -55,11 +55,11 @@ void Hachiko::Scripting::CrystalExplosion::OnStart()
 
 void Hachiko::Scripting::CrystalExplosion::OnUpdate()
 {
-	ResourceAnimation* resource_animation = _explosion_crystal->GetComponent<ComponentAnimation>()->GetCurrentAnimation();
+	ComponentAnimation* resource_animation = _explosion_crystal->GetComponent<ComponentAnimation>();
```

Comment:
```suggestion
	ComponentAnimation* component_animation = _explosion_crystal->GetComponent<ComponentAnimation>();
```

Reply:
Added just a small change, but you have the approval already. Thanks!

Reply:
Perfect lo fixe en otra pr en dev esta correcto mersi